connectors/KnowledgeLakeRPA/readme.md

```diff
@@ -0,0 +1,25 @@
+﻿
+## KnowledgeLake RPA
+KnowledgeLake RPA allows you to integrate any Windows, browser, or legacy system with any other application, data source, or custom API without having to make changes to your application or requiring cooperation from the application’s vendor.
+
+
+## Pre-requisites
+* A Microsoft PowerApps or Microsoft Flow plan with custom connector feature
+* An Azure subscription
+* The Power platform CLI tools
+
+
+## API documentation
+Detailed documentation is located here https://docs.microsoft.com/en-us/connectors/knowledgelake/
+
+## Supported Operations
+Calls a KnowledgeLake RPA Process
+
+
+## How to get credentials
+You must have a KnowledgeLake Subscription 
+
+
+## Deployment instructions
+Please use [these instructions](https://docs.microsoft.com/en-us/connectors/custom-connectors/paconn-cli) to deploy this connector as custom connector in Microsoft Flow and PowerApps
+
```

Comment:
@SteveCurran Please change "Microsoft Flow and PowerApps" to "Microsoft Power Automate and Power Apps"